src/TextEditor.js

```diff
@@ -412,7 +412,6 @@ export default class TextEditor extends Component {
             tabIndex={this.props.tabIndex}
           />
         </div>
-        <Modals />
```

Comment:
@adamsas question for you.. Where is the ```<Modals />``` now since it's removed from here

Reply:
@ThinkTankShark We discussed this with @isuttell, and the <Modals /> should only appear once in a project.

The issue was that multiple <TextEditor /> components would each show the same Modals that are opened. Therefore, it had to be moved out of here. <Modals /> is not part of the example instead.